src/main/java/com/digirati/taxonomy/manager/lookup/normalisation/TextNormaliser.java

```diff
@@ -57,24 +60,48 @@ private void initialisePipeline(String languageKey, String languageName) {
     }
 
     /**
-     * Applies the NLP pipeline to a piece of input text to produce a normalised version of it. This
-     * includes tokenisation, sentence splitting, part of speech tagging, and lemmatisation (if this
-     * is supported for the language in question).
+     * Applies the NLP pipeline to a piece of input text to produce a list of the {@link
+     * Word}s it contains (i.e. all words, excluding stopwords). The NLP pipeline includes
+     * tokenisation, sentence splitting, part of speech tagging, and lemmatisation (if this is
+     * supported for the language in question).
      *
-     * @param text a non-normalised piece of text
-     * @return the normalised form of the input text
+     * @param text a piece of text from which to extract the content words
+     * @return the list of content words contained in the input text
      */
-    public String normalise(String text) {
-        StringBuilder normalisedText = new StringBuilder();
+    public List<Word> extractContentWords(String text) {
         CoreDocument document = new CoreDocument(text);
         pipeline.annotate(document);
-        document.tokens().stream()
+        return document.tokens().stream()
                 .filter(this::isContentWord)
-                .forEach(coreLabel -> normalisedText.append(coreLabel.lemma()).append(" "));
-        return normalisedText.toString().trim();
+                .map(Word::new)
+                .collect(Collectors.toList());
     }
 
     private boolean isContentWord(CoreLabel label) {
         return !stopwords.contains(label.originalText()) && !stopwords.contains(label.lemma());
```

Comment:
I've added in stopword normalisation for now - and if, as discussed, we use only pre-normalised stopword lists we'll be able to drop it. Thinking about it again though, I don't think we'll be able to realistically support all languages with static lists (it would be a pain to to raise a PR to add a new list then wait for a review and a deploy every time someone wants to use this with a new language, and some use cases might want more granular control over this than others) so I think we should probably let the user define their own stopword lists, in which case stopword normalisation definitely makes sense.